src/asset/modules/statusCheck.js

```diff
@@ -0,0 +1,12 @@
+// status checkbox
+const statusCheckBox = (e, ListContent) => {
+  const list = e.target.parentNode.parentNode;
```

Comment:
due to DRY practices, I suggest to use `e.target.closest()` instead of `e.target.parentNode.parentNode`